x-pack/plugins/observability_solution/observability_logs_explorer/common/url_schema/url_schema_v1.ts

```diff
@@ -8,8 +8,12 @@
 import { availableControlsPanels, datasetSelectionPlainRT } from '@kbn/logs-explorer-plugin/common';
 import * as rt from 'io-ts';
 
-export const columnRT = rt.intersection([
+const allowedNames = rt.union([rt.literal('content'), rt.literal('resource')]);
```

Comment:
Nit: Schould we name this `allowedNamesRT` to be consistent with the other types in the file?